content/de/includes/partner-script.js

```diff
@@ -0,0 +1,1609 @@
+;(function () {
```

Comment:
Unless you're planning on translating all partners, it is better to have this file removed. When not present in the German translation it will use the `content/en/includes/partner-script.js` file which may be better initially.